content/docs/guides/components/tftraining.md

```diff
@@ -648,3 +648,6 @@ Events:
  	   * The docker image doesn't exist or can't be accessed (e.g due to permission issues)
 1. If the containers start; check the logs of the containers following the instructions
    in the previous section.
+
+## More information
+* See how to [run a job with gang-scheduling](https://www.kubeflow.org/docs/guides/job-scheduling).
```

Comment:
The link here needs to be relative, so it doesn't break when viewed from a preview site. So, instead of this:
(https://www.kubeflow.org/docs/guides/job-scheduling)

Use this:
(/docs/guides/job-scheduling)